.gitignore

```diff
@@ -2,4 +2,6 @@ node_modules/
 public/app.js
 npm-debug.log
 .env
-.vscode/
\ No newline at end of file
+.vscode/
+z3
```

Comment:
Why are you removing the z3 binary from the repo? Nothing will work otherwise...